javascript/javascript3/week1/index.js

```diff
@@ -0,0 +1,115 @@
+
+//ex-2---- this api will return Array of objects & we can access objet by index like characterData.quotes/episodes/deathes etc
+fetch("https://www.breakingbadapi.com/api/")
+  .then((response) => response.json())
+  .then((characterData) => {
+    console.log(characterData);
+  });
+
+//ex-3 weather app------
+const search_button = document.getElementById("searchButton");
+search_button.addEventListener('click',weatherBycity);
+const location_button = document.getElementById("getLocation");
+location_button.addEventListener('click',weatherBylocation);
+function weatherBycity()
+{
+  if(!document.getElementById("searchCity").value)
+  {
+      alert ("Please enter City name");
+      return false;
+  }
+  const search_city = document.getElementById("searchCity").value;
+  key = '82b892157b6437857ba417dcafa072c1';
+  url='https://api.openweathermap.org/data/2.5/weather?q='+search_city+'&appid='+key;
+  getWeather(url);
+}
+function weatherBylocation()
```

Comment:
code formatting makes the code hard to read. Please install .prettier and submit in a more readable format.

Reply:
Thanks Aleks for reviewing my hw.. I will do...